components/serializer.rst

```diff
@@ -116,29 +116,6 @@ The first parameter of the :method:`Symfony\\Component\\Serializer\\Serializer::
 is the object to be serialized and the second is used to choose the proper encoder,
 in this case :class:`Symfony\\Component\\Serializer\\Encoder\\JsonEncoder`.
 
-Ignoring Attributes when Serializing
```

Comment:
It just [moved down](https://github.com/symfony/symfony-docs/pull/4675/files#diff-c4305609c3522893ccdcd97d049d13e5R258).